numpy/core/_add_newdocs.py

```diff
@@ -1280,7 +1280,7 @@
 
 add_newdoc('numpy.core.multiarray', 'fromstring',
     """
-    fromstring(string, dtype=float, count=-1, sep='', *, like=None)
+    fromstring(string, dtype=float, count=-1, *, sep, like=None)
```

Comment:
`fromstring` is currently a bit of an odd one. It does have a default value for `sep`, 
but it is deprecated, this leaving us with a mandatory parameter right in the middle of a function signature, 
As a compromise I now documented it as a mandatory keyword-only parameter.

Any thoughts here?

Reply:
When was it deprecated?

Reply:
By the looks of it in 1.14:
https://numpy.org/doc/stable/reference/generated/numpy.fromstring.html

Reply:
> Mandatory seems reasonable. Maybe it should get a better default value?

I imagine that `" "` would be the most obvious candidate here.